tomls/omnibus-sepolia/core.toml

```diff
@@ -0,0 +1,26 @@
+[setting.snx_package]
+
+[setting.owner]
+
+[setting.target_preset]
+defaultValue = "main"
+
+[setting.salt]
+defaultValue = "snax"
+
+[setting.bundleSalt]
+defaultValue = "snax"
+
+[clone.oracle_manager]
```

Comment:
Do we need trusted multicall as well here somewhere?

Reply:
It's not really necessary. If we were starting over then. Yeah it probably would be best to just have it on the top level, but it's not going to hurt anything and I don't foresee it hurting anything in the future because of the way that it's deployed. Basically functions like an import.